stock_inventory_valuation_pivot/views/stock_account_views.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="view_inventory_valuation_search" model="ir.ui.view">
+        <field name="model">stock.valuation.layer</field>
+        <field name="inherit_id" ref="stock_account.view_inventory_valuation_search" />
+        <field name="arch" type="xml">
+            <field name="product_tmpl_id" position="after">
+                <field
+                    string="Date Before"
+                    name="create_date"
+                    filter_domain="[('create_date', '&lt;=', self)]"
```

Comment:
As the field is datetime, it is easier to filter records prior to a specific day at 00:00:00
```suggestion
                    filter_domain="[('create_date', '&lt;', self)]"
```